.dockerignore

```diff
@@ -0,0 +1,18 @@
+debug
```

Comment:
Yeah, good call on ``.dockerignore``. 

We do want to exclude as much as possible to speed up the context transfer and as such, the build itself.

To be on the safe side we should also exclude ``.tox`` which can be very large as well (although not sure if it will be there when we run the docker image build, but still).

Reply:
And probably also good idea to add ``node_modules`` to be on the safe side.